test/exchange-files.js

```diff
@@ -146,7 +157,7 @@ describe('exchange files', function () {
         })
       }))
 
-      if (isWindows()) { return }
+      if (isWindows) { return }
       // TODO fix dir tests on Windows
```

Comment:
Is this still a problem that needs fixed? Is there an issue we can link for this?